payments/admin.py

```diff
@@ -1,6 +1,29 @@
+import json
+
 from django.contrib import admin
-from payments.models import StripeAccount, StripeEvent
+from .models import StripeAccount, StripeEvent
+
+
+class StripeAccountAdmin(admin.ModelAdmin):
+
+    readonly_fields = (
+        'user', 'account_id', 'secret_key', 'public_key',
```

Comment:
Do we need to see the `secret_key` values here? Or can we keep it secret such that it requires connecting to the database to see it?